.github/config.yml

```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-linkerd/master/.github/welcome/Layer5-celebration.svg)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/meshery-celebration-dark.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
```

Comment:
Double check the path for the directory, we're in `meshery-linkerd`

Reply:
thank you , I forgot bout these . Fixed it now